src/config.rs

```diff
@@ -384,6 +408,24 @@ fn parse_color_string(color_string: &str) -> Option<ansi_term::Color> {
     predefined_color
 }
 
+fn get_palette<'a>(
+    palettes: &'a HashMap<String, Palette>,
+    palette_name: Option<&str>,
+) -> Option<&'a Palette> {
+    if let Some(palette_name) = palette_name {
+        let palette = palettes.get(palette_name);
+        if palette.is_some() {
+            log::trace!("Found color palette: {}", palette_name);
+        } else {
+            log::debug!("Could not find color palette: {}", palette_name);
```

Comment:
I think `warn` is still more appropriate here. If anything, I was thinking of to (if possible) reduce the number of times this code path is reached.